includes/class-authors.php

```diff
@@ -339,7 +339,7 @@ public function save_title( $data, $postarr ) {
 		$last_name   = ( ! empty( $_POST['last_name'] ) ) ? sanitize_text_field( wp_unslash( $_POST['last_name'] ) ) : get_post_meta( $postarr['ID'], 'last_name', true );
 		$author_name = "{$first_name} {$last_name}";
 
-		if ( '' !== $author_name ) {
+		if ( ' ' !== $author_name ) {
```

Comment:
😲

Reply:
ah I guess `!empty($author_name)` would be much better here

Reply:
With a hard-wired space?? `$author_name = "{$first_name} {$last_name}";`

Reply:
ahh overlooked the space  `!empty(trim($author_name))` will handle the case

Reply:
Done.